tycho-core/src/main/java/org/eclipse/tycho/core/osgitools/EquinoxResolver.java

```diff
@@ -202,6 +202,16 @@ protected ModuleContainer newState(DependencyArtifacts artifacts, Properties pro
             throws BundleException {
         ModuleContainer[] moduleContainerAccessor = new ModuleContainer[1];
         ModuleContainerAdaptor moduleContainerAdaptor = new ModuleContainerAdaptor() {
+
+            @Override
+            public String getProperty(String key) {
+                switch (key) {
+                case "equinox.resolver.revision.batch.size":
```

Comment:
Maybe we should add a comment /reference to bug to explain why this is necessary?